internal/pubsub/mempubsub/mem.go

```diff
@@ -91,7 +70,15 @@ func (t *topic) SendBatch(ctx context.Context, ms []*driver.Message) error {
 func (*topic) IsRetryable(error) bool { return false }
 
 // As implements driver.Topic.As.
-func (t *topic) As(i interface{}) bool { return false }
+// It supports *topic for NewSubscription (see below).
```

Comment:
Can you expand this a bit? Something like:

```
// It supports *topic so that NewSubscription can recover a *topic
// from the concrete type (see below). External users won't be able
// to use As because topic isn't exported.
```

Reply:
Done.